src/main/java/io/chrislowe/discordle/game/GameService.java

```diff
@@ -36,6 +40,8 @@ public SubmissionOutcome submitGuess(String guildId, String discordId, String gu
             return SubmissionOutcome.GAME_UNAVAILABLE;
```

Comment:
I think now with this change that the game being null should be an error

Reply:
Done